devel/forge-devel: New port

yzgyyang opened the pull request:
Work with Git forges from the comfort of Magit (Development Version).

Forge allows you to work with Git forges, such as Github and Gitlab,
from the comfort of Magit and the rest of Emacs.

Forge fetches issues, pull-requests and other data using the forge's
API and stores that in a local database. Additionally it fetches the
pull-request references using Git. Forge implements various features
that use this data but the database and pull-request refs can also be
used by third-party packages.

This is development version of forge and works with devel/magit-devel.

PR:		255962
Approved by:	lwhsu (mentor)

lwhsu commented:
this update devel/forge to 0.2.1 https://bugs.freebsd.org/bugzilla/show_bug.cgi?id=256662
maybe after new snapshot is out?

yzgyyang commented:
@lwhsu this is now updated, and is ready for review.

Commit:
devel/forge-devel: New port

Work with Git forges from the comfort of Magit (Development Version).

PR:		255962
Approved by:	lwhsu (mentor)
